src/main/plugin/iso19139.ca.HNAP/loc/fre/strings.xml

```diff
@@ -60,8 +60,8 @@
   <xsdErrorIs>L'erreur est : </xsdErrorIs>
   <inElement> contenu dans </inElement>
   <enum1>La valeur </enum1>
-  <enum2> n'est pas valide pour le champ </enum2>
-  <enum3>Les valeurs possibles sont : </enum3>
+  <enum2> n'est pas valide pour le champ</enum2>
```

Comment:
If you remove the space at the end, I believe this will cause an issue with the field that is appended after the text?

Reply:
No. I am fixing an issue instead. The xsl calling enum will add extra space. It will be double spaced with this space as well. So I have to remove it.